lib/http/api.go

```diff
@@ -33,6 +33,8 @@ const (
 
 	// AEList lists all datasets in your collection
 	AEList APIEndpoint = "/list"
+	// AECollectionGet returns info on a head dataset in your collection
+	AECollectionGet APIEndpoint = "/collection/get"
```

Comment:
Okay, so, wanted to get some feedback on this endpoint name.

Right now, it would be the only endpoint prefixed with `/collection`, since the main collection endpoint is just `/list`

Some other ideas:
`/header`
`/details`
`/getfromcollection`

Reply:
See my earlier comment about the method name for the same answer here (spoiler: `/collection/get` is good)

Reply:
👍 going to keep for now, we can change in a future pr if we rename to something we find more sensible